src/Tasks/ContentReviewEmails.php

```diff
@@ -15,15 +16,17 @@
 use SilverStripe\SiteConfig\SiteConfig;
 use SilverStripe\View\ArrayData;
 use SilverStripe\View\SSViewer;
-use SilverStripe\ContentReview\Models\ContentReviewLog;
 
 /**
  * Daily task to send emails to the owners of content items when the review date rolls around.
  */
 class ContentReviewEmails extends BuildTask
 {
+    private array $invalid_emails = [];
+
     /**
      * @param HTTPRequest $request
+     * @throws Exception
      */
     public function run($request)
     {
```

Comment:
Probably right at the top here (just after `ContentReviewCompatability::start();`) is when we should be validating the `$siteConfig->ReviewFrom` address.

Reply:
DONE